test/images/README.md

```diff
@@ -0,0 +1,117 @@
+# Kubernetes test images
+
+## Overview
+
+All the images found here are used in Kubernetes tests that ensure its features and functionality.
+The images are built and published as manifest lists, allowing multiarch and cross platform support.
+
+This guide will provide information on how to: make changes to images, bump their version, build the
+new images, test the changes made.
+
+
+## Prerequisites
+
+In order to build the docker test images, a Linux node is required. The node will require `make`
```

Comment:
pretty sure these also build on macOS?

Reply:
I was able to use Docker for Mac for agnhost:2.1 - 2.3, though I had to comment out https://github.com/kubernetes/kubernetes/blob/master/test/images/image-util.sh#L91 

I have been able to get away with this because Docker for Mac provides `binfmt_misc` support out of the box https://docs.docker.com/docker-for-mac/multi-arch/